Firebase/InstanceID/FIRInstanceIDBackupExcludedPlist.m

```diff
@@ -104,14 +108,9 @@ - (NSDictionary *)contentAsDictionary {
   return self.cachedPlistContents;
 }
 
-- (void)moveToApplicationSupportSubDirectory {
-  self.fileInApplicationSupport =
-      [self moveToApplicationSupportSubDirectory:self.applicationSupportSubDirectory];
-}
-
 - (BOOL)moveToApplicationSupportSubDirectory:(NSString *)subDirectoryName {
```

Comment:
I think you'll need to expose this in the header (or in the tests via a category)